### DISCUSSION TITLE
Welcome to Stable-Diffusion-Unity-Integration Discussions!

### DISCUSSION THREAD

**Discussion by dobrado76** [dobrado76/Stable-Diffusion-Unity-Integration#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satriagilang80** [comment]
is possible to access this "alwayson_scripts": {} ?
